core/templates/pages/contributor-dashboard-page/question-opportunities/question-opportunities.component.spec.ts

```diff
@@ -138,9 +139,10 @@ describe('Question opportunities component', function() {
 
   it('should open requires login modal when trying to select a question and' +
     ' a skill difficulty and user is not logged', function() {
-    spyOn(userService, 'getUserInfoAsync').and.returnValue($q.resolve({
-      isLoggedIn: () => false
-    }));
+    spyOn(userBackendApiService, 'getUserInfoAsync').and.returnValue(
+      $q.resolve(
+        {isLoggedIn: () => false
+        }));
```

Comment:
Indentation is a bit awkward. Can you the same formatting used in `core/templates/pages/contributor-dashboard-page/translation-opportunities/translation-opportunities.component.spec.ts`.

```suggestion
      $q.resolve({
        isLoggedIn: () => true
      }));
```

Ditto with instances below.